test/test_helper.rb

```diff
@@ -1,5 +1,7 @@
 root_location = File.expand_path("../../", __FILE__)
 inc_dirs = Dir.glob("#{root_location}/library/*/src")
+# Needed to test Y2DIR support in Yast::Directory
+inc_dirs << "#{root_location}/library/general/test"
```

Comment:
Yes, that was my first attempt and it works if you do ```rspec directory_test.rb```. But it does not work with ```rake test:unit```, since all the tests are called in a single command so you cannot set Y2DIR in the tests in a reliable way.